mod_ci/controllers.py

```diff
@@ -1290,8 +1281,8 @@ def progress_type_request(log, test, test_id, request) -> bool:
 
             for p in times:
                 parts = p.time.split(',')
-                start = datetime.datetime.strptime(parts[0], '%Y-%m-%d %H:%M:%S')
-                end = datetime.datetime.strptime(parts[-1], '%Y-%m-%d %H:%M:%S')
+                start = datetime.datetime.strptime(parts[0], '%Y-%m-%d %H:%M:%S.%f')
+                end = datetime.datetime.strptime(parts[-1], '%Y-%m-%d %H:%M:%S.%f')
```

Comment:
@thealphadollar I found this bug, it might not be relevant to this issue but the timestamps are stored in this format in the DB, I wrote the test for this...

Reply:
Thanks, good catch 👍🏽